must-call-checker/src/main/java/org/checkerframework/checker/mustcall/Stream.astub

```diff
@@ -132,6 +113,34 @@ class BufferedInputStream extends FilterInputStream {
     void close() throws IOException;
 }
 
+class FileOutputStream extends OutputStream {
```

Comment:
I think these stubs for `FileOutputStream`, `FileInputStream`, and `RandomAccessFile` conflict with #266, which has fewer extraneous methods and comments that explain why the constructors are treated differently.